### PR TITLE
add cssnano

### PR DESCRIPTION
**Pull request by amirfefer**
Supported Versions:

 * Nightly

### CODE REVIEW

File: packages/foreman/nodejs-cssnano/nodejs-cssnano.spec [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+rm -rf %{buildroot} %{npm_cache_dir}
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+%license node_modules/%{npm_name}/LICENSE-MIT
+%doc node_modules/%{npm_name}/CHANGELOG.md
+%doc node_modules/%{npm_name}/README.md
+
+%changelog
+* Wed Jun 02 2021 vagrant 4.1.10-1
```
Comment (tbrisker) [inline packages/foreman/nodejs-cssnano/nodejs-cssnano.spec:366]: The name is wrong but i don't think this is the reason the lint failed to run

**Comment by tbrisker** [comment]
you need to add the package in `package_manifest.yaml`, `comps/comps-foreman-rhel7.xml`, `rel-eng/tito.props` and `comps/comps-foreman-el8.xml`

**Comment by amirfefer** [comment]
@tbrisker , thanks 👍 
Added

**Review by tbrisker — APPROVED** [review]

**Review by evgeni — APPROVED** [review]
I took the liberty to also add it to el8 tito lists

**Comment by evgeni** [comment]
```console
% koji add-pkg foreman-nightly-el8 nodejs-cssnano --owner=kojiadm
Adding 1 packages to tag foreman-nightly-el8
% koji add-pkg foreman-nightly-rhel7 tfm-nodejs-cssnano --owner=kojiadm
Adding 1 packages to tag foreman-nightly-rhel7
```

**Pull request merged**